windows-driver-docs-pr/install/difx-guidelines.md

```diff
@@ -8,15 +8,15 @@ ms.localizationpriority: medium
 
 # DIFx Guidelines
 
-Starting in Windows 10 Version 1607 (Redstone 1), the Driver Install Frameworks (DIFx) tools are no longer included in the WDK.  Instead, we recommend providing your driver as a standalone package that doesn't require an installer, ideally through Windows Update.  To add your driver to Windows Update, the first step is to submit your driver package to the [Windows Hardware Dev Center](https://partner.microsoft.com/dashboard).
+Starting in Windows 10 Version 1607 (Redstone 1), the Driver Install Frameworks (DIFx) tools (`Difxapi.dll`, `Difxapp.dll`, `Difxappa.dll`, and `DPInst.exe`) are deprecated and are no longer included in the WDK.  Instead, we recommend providing your [driver pacakge](https://docs.microsoft.com/windows-hardware/drivers/install/driver-packages) as a standalone driver package that does not require an installer.  A standalone driver package is a driver package that is self contained and itself adds any settings or configuration required for the functioning of the driver package and the devices it installs on instead of an installer seperately modifying additional system state as part of adding the driver package to the system.  Standalone driver packages are required in order to have robust support of all driver package scenarios.  Ideally, you would publish this standalone driver package to be distributed through Windows Update.  To publish your driver to Windows Update, the first step is to submit your driver package to the [Windows Hardware Dev Center](https://partner.microsoft.com/dashboard).
```

Comment:
@ jasonknichel - when you say you need the standalone to have robust support all pkg scenarios -- could we name a key one or two for which they would miss out on support if they don't go standalone?

Reply:
Some examples (though not limited to these examples):
If it is not standalone, it 
1) cannot be published on windows update and successfully installed on machines.
2) cannot be added to an image offline and work correctly.

Reply:
Thanks, just added those to the page.  Should appear on the live site in a few minutes